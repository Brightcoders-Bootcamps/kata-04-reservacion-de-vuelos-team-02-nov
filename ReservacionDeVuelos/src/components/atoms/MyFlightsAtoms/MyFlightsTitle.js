import React from 'react';
import {View, Text} from 'react-native';
import {styles} from '../../../styles/MyFlightsStyles';


const MyFlightsTitle = ({title}) => {
  return (
    <View style={styles.titleContainer} testID='view'>
      <Text style={styles.myFlightsTitle} testID='text'>
        {title}
      </Text>
    </View>
  );
};

export default MyFlightsTitle;