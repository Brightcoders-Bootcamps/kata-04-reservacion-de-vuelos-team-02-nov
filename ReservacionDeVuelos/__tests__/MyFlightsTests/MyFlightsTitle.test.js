import React from 'react'
import {shallow} from 'enzyme'
import MyFlightsTitle from '../../src/components/atoms/MyFlightsAtoms/MyFlightsTitle'

const props = {
    title: 'title'
}

describe('Render correctly', () =>{
    const wrapper = shallow(<MyFlightsTitle {...props}/>)
    test('snapshot', () => {
        expect(wrapper).toMatchSnapshot();
      });
    test('Render correctly', () => {
        const render = wrapper.findWhere((node) => node.prop('testID') === 'view');
        expect(render.exists()).toBeTruthy();
      });
      test('Title correctly', () => {
        const title = wrapper.findWhere((node) => node.prop('testID') === 'text');
        expect(title.props().children).toEqual(props.title);
      });
      
})