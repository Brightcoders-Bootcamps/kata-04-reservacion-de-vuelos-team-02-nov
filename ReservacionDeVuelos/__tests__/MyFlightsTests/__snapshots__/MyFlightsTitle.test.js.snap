// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly snapshot 1`] = `
<View
  style={
    Object {
      "margin": 10,
    }
  }
  testID="view"
>
  <Text
    style={
      Object {
        "color": "#6472ee",
        "fontSize": 35,
        "fontWeight": "bold",
      }
    }
    testID="text"
  >
    title
  </Text>
</View>
`;
